app/model/Google/OAuth.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Google;
+
+use DateTimeImmutable;
+use Doctrine\ORM\Mapping as ORM;
+use Model\Common\Aggregate;
+use Model\Common\UnitId;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="google_oauth", uniqueConstraints={@ORM\UniqueConstraint(name="unitid_email", columns={"unit_id", "email"})} )
```

Comment:
Je ten unique constraint kontrolovaný i někde v kódu? Takhle totiž při pokusu o připojení už existujícího účtu uživatel dostane 500.

Reply:
to je v te sekci, ze se nejdriv kontroluje jestli uz propojeni neexistuje viz GoogleRepository::saveAuthCode

Reply:
Ok

Reply:
Dík :+1: